### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by ramidzkh**
## Changes
* Uses the languages files more
* Fixes toggling of void and auto-pickup
* Move auto-pickup Mixin to be applied on both clients and dedicated servers
* Adds all the tiers to the item group
* Optimize a little
---
I tried adding a block-place dispense logic to buffers but buffer blocks are quite buggy (middle clicking and placing one can crash your game).

### CODE REVIEW

**Review by vini2003 — COMMENTED** [review]
Seems good, though any reason `public static String PICKUP_RETRIEVER = "pickup";` is not `final`?

**Comment by ramidzkh** [comment]
> Seems good, though any reason `public static String PICKUP_RETRIEVER = "pickup";` is not `final`?

Must have forgotten to make it final

File: src/main/java/buffer/registry/NetworkRegistry.java [hunk 1]
```diff
@@ -76,14 +76,15 @@ public static void registerPackets() {
         });
 
         ServerSidePacketRegistry.INSTANCE.register(BUFFER_PICKUP_PACKET, (packetContext, packetByteBuffer) -> {
+            boolean something = packetByteBuffer.readBoolean();
             packetContext.getTaskQueue().execute(() -> {
                 PlayerEntity playerEntity = packetContext.getPlayer();
                 if (playerEntity.inventory.getMainHandStack().getItem() == ItemRegistry.BUFFER_ITEM) {
                     BufferInventory bufferInventory = BufferInventory.fromTag(playerEntity.inventory.getMainHandStack().getTag());
-                    bufferInventory.isPickup = packetByteBuffer.readBoolean();
+                    bufferInventory.isPickup = something;
```
Comment (vini2003) [inline src/main/java/buffer/registry/NetworkRegistry.java:82]: I love this hehe. Will solve locally.

File: src/main/resources/assets/buffer/lang/en_us.json [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
-    "item.buffer.buffer": "Buffer",
-    "block.buffer.buffer": "Buffer"
-}
\ No newline at end of file
+  "item.buffer.buffer": "Buffer",
+  "block.buffer.buffer": "Buffer",
+  "buffer.gui.pickup": "Collect",
+  "buffer.gui.void": "Void",
+  "buffer.tooltip.tier": "§6§lTier %s",
+  "buffer.tooltip.pickup.true": "§9Collect:§7§o Yes",
```
Comment (vini2003) [inline src/main/resources/assets/buffer/lang/en_us.json:7]: Good call with _Yes_ / _No_, not sure how I didn't think of it before.

**Comment by vini2003** [comment]
Done, thank you!

**Pull request merged**